src/main/java/Containers/StorageLoadResult.java

```diff
@@ -0,0 +1,15 @@
+package Containers;
+
+import Collections.List;
+
+public class StorageLoadResult {
```

Comment:
making public fileds is a bad idea. 

in case you tied of boiler code, look at Lombok, it will help with these